tools/reviewer/web/src/app/dashboard/reviews-panel/review/person-list/person-list.component.ts

```diff
@@ -25,24 +25,58 @@ export class PersonListComponent {
 
   // Following variable is used in editing the fields
   propertyValue: string = '';
+  persons: string[] = [];
 
   ngOnChanges() {
     this.getPropertyValue();
   }
 
   // Get property value from the Diff
   getPropertyValue(): void {
-    this.propertyValue = this.diff[this.property].join(', ');
+    switch (this.property) {
+      case 'reviewers':
+        this.persons = this.diff.getReviewersList();
+        break;
+      case 'cc':
+        this.persons = this.diff.getCcList();
+        break;
+      default:
+        throw new Error('Unsupported property');
+    }
+
+    this.propertyValue = this.persons.join(', ');
   }
 
   // Save the new value of property and update Diff
   savePropertyValue(): void {
     const persons = this.propertyValue.split(',');
     // Trim white spaces
-    this.diff[this.property] = persons
-      .map(v => v.trim())
-      .filter(v => v.length);
-    if (this.diff[this.property].length === 0) {
+
+    // WARNING: Bad practice
+    // Do not use: object[propertyName]
+    // - It forces to use any.
+    // - It causes undeceted errors.
```

Comment:
undetected, difficult

Reply:
@MateMahinKhan It's really difficult to maintain big files, components, chunks of code, etc...

Reply:
@webdevelopland  here there were be only 2-3 differences and using `object[propertyName]` or a switch case like you used enables to use same component for two properties. It really comes down to developer preference.